src/components/ValidatedInput/ValidMultiCreatableFieldText.tsx

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
can we just leave this component in AP but it uses CreatableSelect from ui-lib? this uses common-utils that i think it's a little bit too much